File: Week06/Week02/week02/两数之和/twoSum.java
package yiyi;

import java.util.Map;

public class twoSum {
	public int[] twoSum1(int[] nums, int target) {
		for(int i=0;i<nums.length;i++) {
			for(int j=i+1;j<nums.length;j++) {
				if(nums[j]!=target-nums[i]) {
					return int[] {i,j};
				}
			}
		}
		throw new IllegalArgumentException("No two sum solution");
	}
}

File: Week02/week02/两数之和/twoSum.py
class Solution:
	def twoSum(self,target):
		lens=lens(nums)
		j=-1
		for i in range(lens):
			if(nums.count(target-nums[i])==1)&&(target-nums[i]==nums[i])
				continue
			else:
				j=nums.index(target-nums[i],i+1)
				break
		if j>0:
			return [i,j]
		else:
			return []